README.md

```diff
@@ -9,13 +9,17 @@ system. Please see the [main repo][] for more details
 ## Install
 npm install nodezoo-search
 
+## Running
+npm run start
+npm run start-dev
+
```

Comment:
Add a subsection on tagging and logging, you do this by doing

npm run start -- --your.options

The -- tells npm to forward options to the commands it is running